public/app/features/provisioning/hooks/index.ts

```diff
@@ -1,7 +1,2 @@
-export { useCreateOrUpdateRepository } from './useCreateOrUpdateRepository';
-export { useCreateOrUpdateRepositoryFile } from './useCreateOrUpdateRepositoryFile';
-export { useGetResourceRepository } from './useGetResourceRepository';
-export { useIsProvisionedNG } from './useIsProvisionedNG';
-export { usePullRequestParam } from './usePullRequestParam';
-export { useRepositoryJobs } from './useRepositoryJobs';
-export { useRepositoryList } from './useRepositoryList';
+// Empty file index -- betterer does not want us to reexport hooks from another file
+export const dummy = 10;
```

Comment:
@Clarity-89 obviously this is silly... but if i remove the file the build fails 🙃

Reply:
This file shouldn't be here, probably got messed up when merging to main.

Reply:
There were a few index imports in left the code, so that's why the build was failing.